pcieutil/main.py

```diff
@@ -131,7 +125,7 @@ def check():
 def generate():
     '''Generate config file with current pci device'''
     platform_pcieutil.dump_conf_yaml()
-    click.echo("Generate config file pcie.yaml under path %s" % platform_plugins_path)
+    click.echo("Generate config file pcie.yaml under path %s" % platform_path)
```

Comment:
Suggest to change to `click.echo("Generated config file '{}/pcie.yaml'".format(platform_path))`